packages/gestalt/src/TypeaheadInputField.css

```diff
@@ -0,0 +1,16 @@
+
+.icon {
+  composes: marginEnd2 paddingX1 paddingY1 from "./boxWhitespace.css";
+  composes: noBorder from "./Borders.css";
+  composes: pointer from "./Cursor.css";
+  composes: right0 from "./Layout.css";
+  composes: transparentBg from "./Colors.css";
+  outline: none;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.inputRadius {
+  composes: rounding4 from './Borders.css'
+}
```

Comment:
@simkessy NIT could we
* Remove the first empty line in this file
* Add an extra empty line add the end of the file

![image](https://user-images.githubusercontent.com/127199/87581665-2cf30080-c68e-11ea-8bc2-ca796e171dff.png)